src/main/java/com/microsoft/azure/kusto/kafka/connect/sink/KustoSinkConfig.java

```diff
@@ -102,6 +105,12 @@ public class KustoSinkConfig extends AbstractConfig {
     private static final String KUSTO_SINK_ENABLE_TABLE_VALIDATION_DOC = "Enable table access validation at task start.";
     private static final String KUSTO_SINK_ENABLE_TABLE_VALIDATION_DISPLAY = "Enable table validation";
 
```

Comment:
Quick q :  Would the following simplify this ?
```new ObjectMapper().configure(JsonParser.Feature.ALLOW_SINGLE_QUOTES, true);```

Reply:
Indeed, I was under the impression that `enable` was returning `void`, but no! Changed it to simplify